mobly/test_runner.py

```diff
@@ -377,6 +383,13 @@ def get_info(objects):
             raise signals.ControllerError(
                 "Controller module %s did not return a list of objects, abort."
                 % module_ref_name)
+        if min_number is not None:
```

Comment:
Why support None here? I'd prefer to just simplify the code since the default is 1 anyway.